styles/work-section.css

```diff
@@ -146,4 +158,699 @@
+    margin-left: 85px;
+    font-weight: 700;
+  }
+
+  .multi-post-major-div {
+    display: flex;
+    margin-top: 50px;
+    margin-bottom: 20px;
+    width: 786;
+    margin-left: 20px;
+  }
+
+  #desktop-yoga-image {
+    display: block;
+    width: 450px;
+    height: 300px;
+  }
+
+  .desktop-project-div {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    width: 740px;
+    align-self: center;
+    margin-bottom: 50px;
+  }
+
+  #multi-post-title {
+    margin-top: 30px;
+    font-family: 'Crete Round', monospace;
+    font-size: 20px;
+    font-weight: 700;
+    color: #091e42;
+  }
+
+  .multi-post-button {
+    border: 1px solid;
+    border-color: #8993a4;
+    padding: 5px;
+  }
+
+  .work-description {
+    margin: 0;
+  }
+
+  .button-div {
+    height: 35px;
+    margin-top: 0;
+  }
+
+  .floating-1 {
+    display: none;
+  }
+
+  .see-project-button {
+    display: none;
+  }
+
+  .see-project-button-1 {
+    margin-top: 0;
+    width: 235px;
+    background-color: #ff6b00;
+    border: 0;
+    height: 50px;
+    color: white;
+    font-family: Inter, monospace;
+    margin-left: 5px;
+  }
+
+  .first-see-project-button {
+    display: none;
+  }
+
+  .first-see-project-button-desktop {
+    display: flex;
+    width: 100px;
+    padding: 5px;
+    background-color: #ff6b00;
+    border: 0;
+    height: 30px;
+    color: #fff;
+    font-family: Inter, monospace;
+    font-size: 12px;
+    font-weight: bold;
+    margin-top: 35px;
+  }
+
+  .multi-post-div {
+    align-items: flex-start;
+    height: 200px;
+    margin-left: 15px;
+    width: 250px;
+  }
+
+  .mobile-backdrop {
+    display: none;
+    margin: 10px;
+  }
+
+  .project-image {
+    margin-left: 0;
+    width: 235px;
+  }
+
+  .project-title {
+    display: none;
+  }
+
+  .project-description {
+    display: none;
+  }
+
+  .project-img-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    top: 0;
+    height: 226px;
+    width: 235px;
+  }
+
+  .desktop-backdrop {
+    display: flex;
+    position: relative;
+  }
+
+  .floating {
+    display: none;
+  }
+
+  .desktop-floating {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    position: absolute;
+    color: #fff;
+    margin-top: 50px;
+    height: 100px;
+  }
+
+  .individual-project-title {
+    display: flex;
+    font-family: 'Crete Round', monospace;
+    font-size: 15px;
+    line-height: 24px;
+    font-weight: 400;
+    margin-bottom: 20px;
+    width: 200px;
+  }
+
+  .individual-project-title-website {
+    margin-top: 40px;
+  }
+
+  .individual-project-description {
+    display: flex;
+    font-family: Inter, monospace;
+    font-size: 12px;
+    font-weight: 400px;
+    margin-top: 0;
+    width: 180px;
+  }
+
+  .project-1-buttons-ul {
+    height: 30px;
+    margin-top: 0;
+    width: 200px;
+  }
+
+  .project-1-buttons {
+    padding: 5px;
+    font-size: 13px;
+  }
+}
+
+@media (min-width: 1024px) {
```

Comment:
Please, ensure that the mobile version is responsive until the desktop size (768px or higher).

Your design at 746px:
![image](https://user-images.githubusercontent.com/20554583/208044674-2e970b02-be8f-4c46-bcec-ea9a11005055.png)